src/components/UserpdfSelector.js

```diff
@@ -47,12 +49,24 @@ class UserpdfSelector extends Component {
         />
       );
     }
+    if (!isFetching && userpdfs.length < 1) {
+      return <p>No pdfs found.</p>
+    }
+
+    const checkbox = userpdfId ? <DoneUserpdf toggleDoneHandler={this.makeToggleDoneHandler(userpdfs.find(function(p) {
```

Comment:
In general I don't like this if syntax -- it's hard to skim, seems error prone. But not really worth an edit here!